socorro/app/fetch_transform_save_app.py

```diff
@@ -82,7 +82,7 @@ class FetchTransformSaveApp(App):
     )
 
     def __init__(self, config):
-        super(FetchTransformSaveApp, self).__init__(config)
+        super().__init__(config)
```

Comment:
I should do a pass on converting `super()` calls to Python 3 format. It's much easier to deal with.